rackhd_BIST/bist_utils.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+"""
+RackHD build in self test script
```

Comment:
typo: build -> built: Built-in self-test, see https://en.wikipedia.org/wiki/Built-in_self-test